### PR TITLE
Commit entitiy changes to datastore

### PR DESCRIPTION
**Pull request by andreaspsson**

### CODE REVIEW

File: datastore_entity/src/lib.rs [hunk 1]
```diff
@@ -141,6 +150,87 @@ where
...
+        builder
+            .execute()
+            .map_err(|_e: google_datastore1::Error| -> String {
+                "Begin transaction request failed".to_string()
+            })?;
+    let is_insert = !entity.has_key();
+    let key = entity
+        .key()
+        .unwrap_or_else(|| -> Key { generate_empty_key(kind) });
+    let properties: DatastoreProperties = DatastoreProperties::from(entity).unwrap();
```
Comment (bes) [inline datastore_entity/src/lib.rs:192]: Implement this instead?

```rust
impl TryFrom<DatastoreEntity> for Entity {
 ...
}
```

then you can do `entity.try_into().map_err(|_| "some err".to_string())?` instead?

File: datastore_entity/src/lib.rs [hunk 2]
```diff
@@ -141,6 +150,87 @@ where
...
+        }]),
+    }
+}
+
+pub fn commit_one<A>(
+    entity: DatastoreEntity,
+    kind: String,
+    token: A,
+    project_name: &String,
+) -> Result<DatastoreEntity, String>
```
Comment (bes) [inline datastore_entity/src/lib.rs:169]: maybe possible to refactor later that `commit_one` is a special case of `commit_many` with some shared code...

**Review by bes — COMMENTED** [review]
Too many `unwrap` too few `?` and too few `Error` enums... =)

File: datastore_entity/src/lib.rs (hunk 1 above)
Comment (andreaspsson) [reply datastore_entity/src/lib.rs:192]: yes, sounds smart, that I will do!

Comment (andreaspsson) [reply datastore_entity/src/lib.rs:192]: fixed in https://github.com/andreaspsson/datastorers/pull/16

File: datastore_entity/src/lib.rs (hunk 2 above)
Comment (andreaspsson) [reply datastore_entity/src/lib.rs:169]: yes, I think most code can be reused, some thing for get_many/get_one

File: datastore_entity/src/lib.rs (hunk 1 above)
Comment (andreaspsson) [reply datastore_entity/src/lib.rs:192]: fixed in https://github.com/andreaspsson/datastorers/pull/16

**Pull request merged**